solutions/standard/variables.tf

```diff
@@ -61,6 +68,122 @@ variable "secret_manager_tags" {
+}
+
+variable "dns_provider_name" {
+  type        = string
+  description = "Name of the DNS provider for the public_cert secrets engine"
+  default     = "certificate-dns"
+}
+
+variable "ca_name" {
+  type        = string
+  description = "Secret Managers certificate authority name"
+  default     = "cert-auth"
+}
+
+variable "private_key_secrets_manager_secret_id" {
+  type        = string
+  description = "The secret ID of your ACME private key. Required if acme_letsencrypt_private_key is not set. If both are set, this value will be used as the private key."
+  default     = null
+}
+
+variable "private_key_secrets_manager_instance_guid" {
+  type        = string
+  description = "The Secrets Manager instance GUID of the Secrets Manager containing your ACME private key. Required if acme_letsencrypt_private_key is not set."
+  default     = null
+}
+
+variable "private_key_secrets_manager_region" {
+  type        = string
+  description = "The region of the Secrets Manager instance containing your ACME private key. (Only needed if different from the region variable)"
+  default     = null
+}
+
+variable "acme_letsencrypt_private_key" {
+  type        = string
+  description = "The private key generated by the ACME account creation tool. Required if private_key_secrets_manager_instance_guid and private_key_secrets_manager_secret_id are not set."
+  sensitive   = true
+  default     = null
+}
+
+# Private cert engine config
+variable "private_engine_enabled" {
+  type        = bool
+  description = "Set this to true to to configure an IBM Secrets Manager IAM credentials engine for an existing IBM Secrets Manager instance. If set to false, no iam engine will be configured for your secrets manager instance. For more details, see https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-configure-iam-engine."
+  default     = false
+}
+
+variable "private_engine_name" {
+  type        = string
+  description = "The name of the IAM Engine used to configure an IBM Secrets Manager IAM credentials engine for an existing IBM Secrets Manager instance."
+  default     = "private-engine-sm"
+}
+
+variable "root_ca_name" {
+  type        = string
+  description = "Name of the Root CA to create for a private_cert secret engine"
+  validation {
+    condition     = length(var.root_ca_name) >= 2
+    error_message = "length of root_ca_name must be >= 2"
+  }
+  default = "root-ca"
+}
+
+variable "root_ca_common_name" {
+  type        = string
+  description = "Fully qualified domain name or host domain name for the certificate to be created"
+
+  validation {
+    condition     = length(var.root_ca_common_name) >= 4 && length(var.root_ca_common_name) <= 128
+    error_message = "length of root_ca_common_name must be >= 4 and <= 128"
+  }
+
+  validation {
+    condition     = can(regex("(.*?)", var.root_ca_common_name))
+    error_message = "root_ca_common_name must match regular expression /(.*?)/"
+  }
+  default = "terraform-modules.ibm.com"
```

Comment:
we probably need a better default value here, I'm just not sure what it would be. Other option is to default it to `null` but have logic to ensure its not null if `private_engine_enabled` is true. If we can find a good default value, I would prefer that, but it may be unique for every consumer?